io.mjml.mjml-app.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "io.mjml.mjml-app",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "18.08",
+    "command": "mjml-app",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+	"--filesystem=home"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
```

Comment:
These are not needed, runtimes based on 18.08 use their own hardened flags.

Reply:
Which part exactly became unnecessary, the keys under "build-options"?